src/main/java/newbank/client/ExampleClient.java

```diff
@@ -4,19 +4,25 @@
 import java.io.IOException;
 import java.io.InputStreamReader;
 import java.io.PrintWriter;
+import java.io.Reader;
 import java.net.Socket;
 import java.net.UnknownHostException;
 
+import newbank.utils.ConsoleDisplay;
+import newbank.utils.Display;
+
 public class ExampleClient extends Thread {
 
   private Socket server;
   private PrintWriter bankServerOut;
   private BufferedReader userInput;
   private Thread bankServerResponceThread;
+  private Display display = new ConsoleDisplay();
```

Comment:
Perhaps this also should be injected via constructor?

Reply:
Hi Arek,
Good question. I refrained from doing that to keep the number of constructor arguments minimal. Furthermore having it in the constructor would force every client from having to supply it, when it is not essential to the working of the class.

Kindest Regards,
Jules